x-pack/plugin/esql/qa/testFixtures/src/main/resources/string.csv-spec

```diff
@@ -1194,6 +1194,38 @@ a:keyword           | upper:keyword         | lower:keyword
 π/2 + a + B + Λ ºC  | Π/2 + A + B + Λ ºC    | π/2 + a + b + λ ºc
 ;
 
+reverse
```

Comment:
Add more complex csv tests. For example:
- `from employees | sort emp_no | eval jobs_reversed = reverse(job_positions) | keep job*, emp_no | limit 5` which deals with multi-values and `null`s
- `from employees | sort emp_no | eval name_reversed = reverse(reverse(first_name)), eq = name_reversed == first_name | keep emp_no, first_name, name_reversed, eq | limit 5` that uses nested function calls.
- `row x = null | eval y = reverse(x)`
- `row x = 1 | eval y = reverse((null + 1)::string)`

Reply:
For csv tests, I forgot we now have `books` index that also has `text` field type (`employees` and `row` only have `keyword` field types). Please, look into adding couple of tests with `books` index as well. Thanks.

Reply:
Added in the following commits.

https://github.com/elastic/elasticsearch/pull/113297/commits/627144c076a59504a99b695487ced8dbe6703543
https://github.com/elastic/elasticsearch/pull/113297/commits/8f11f095d0b8abf9fe17c015330498156bc75172
https://github.com/elastic/elasticsearch/pull/113297/commits/f5007e7bdb32c78966c64d0efa80aaf292072a59